test/jdk/java/net/Inet6Address/B6206527.java

```diff
@@ -28,7 +28,7 @@
  * @library /test/lib
  * @build jdk.test.lib.NetworkConfiguration
  *        jdk.test.lib.Platform
- * @run main B6206527
+ * @run main/othervm B6206527
```

Comment:
Does it need othervm mode? Best to not use it if it can be avoided.

Reply:
I agree with Michael that the test doesn't need the `/othervm` mode.

Reply:
I included it originally as the issue was related to sockets not being closed and running in `othervm` mitigates negative effects associated with that. However using the try-with-resources prevents this from being the case, so I agree that it is no longer required and will update accordingly.